$OpenBSD: patch-configure,v 1.3 2011/12/04 12:57:09 edd Exp $

MPlayer is not actually a build dependency, so don't bail if not installed at build time.

Bug raised upstream.

--- configure.orig	Fri Nov 25 13:48:50 2011
+++ configure	Sun Dec  4 12:42:05 2011
@@ -3441,9 +3441,9 @@ $as_echo "no" >&6; }
 fi
 
 
-	if test "x${MPLAYER}" = "xno"; then :
-  as_fn_error $? "mplayer is missing" "$LINENO" 5
-fi
+#	if test "x${MPLAYER}" = "xno"; then :
+#  as_fn_error $? "mplayer is missing" "$LINENO" 5
+#fi
 
 	# server just cant work without sqlite
 
